Framework/DataHandling/test/SaveGSSTest.h

```diff
@@ -398,7 +370,10 @@ class SaveGSSTestPerformance : public CxxTest::TestSuite {
 private:
   std::vector<SaveGSS *> saveAlgPtrs;
 
-  const int numberOfIterations = 5;
+  // Controls the speed of the test
+  const int m_numberOfBinsToSave = 20000;
+  // Multiplied by n times specified below
+  const int m_numberOfIterations = 5;
```

Comment:
@DavidFair the major issue with the performance test was this iterative run of the algorithm. When you initially wrote the unit test, there were no iterations. It would be better to get rid of this and just run the algorithm once with a workspace with 100000 bins as per the original implementation.